.gitignore

```diff
@@ -16,6 +16,7 @@ a.out
 build*/
 cmake-build-*/
 doxygen/
+event-formation-unit/
```

Comment:
Why is this needed?

Reply:
The performance tests run using the Centos artefact which is in a folder called event-formation-unit, so building the efu in a folder called "event-formation-unit" instead of "build" means you can run them locally without any changes to the script